llvm/lib/Target/X86/AsmParser/X86AsmParser.cpp

```diff
@@ -3784,6 +3784,68 @@ bool X86AsmParser::processInstruction(MCInst &Inst, const OperandVector &Ops) {
     Inst.setOpcode(X86::INT3);
     return true;
   }
+  // `{evex} cmp <>, <>` is alias of `ccmpt {dfv=of,sf,zf,cf} <>, <>`
+#define FROM_TO(FROM, TO)                                                      \
+  case X86::FROM: {                                                            \
+    if (ForcedOpcodePrefix == OpcodePrefix_EVEX) {                             \
+      Inst.setFlags(~(X86::IP_USE_EVEX) & Inst.getFlags());                    \
```

Comment:
Why this `Inst.setFlags(~(X86::IP_USE_EVEX) & Inst.getFlags());`?

Reply:
Or it will print out wit `{evex} ccmpt ...`

Reply:
Wait a moment. I just noticed the incorrectly resolved comment. You can not cancel a Flag setting by using `Inst.setFlags(~Flag & Inst.getFlags());`

It would affect other flags.

Reply:
why? it's only MI->getFlags(). But not TSFlags.